src/api/bundler-namespace.ts

```diff
@@ -0,0 +1,114 @@
+import {
+  UserOperation,
+  UserOperationEstiamateGasResponse,
+  UserOperationReceipt,
+  UserOperationRequest
+} from '../types/types';
+import { AlchemyConfig } from './alchemy-config';
+
+/**
+ * The Bundler namespace contains methods used for interacting performing user operations and
+ * checking on the state of submitted operations.
+ *
+ * Do not call this constructor directly. Instead, instantiate an Alchemy object
+ * with `const alchemy = new Alchemy(config)` and then access the transact
+ * namespace via `alchemy.paymaster`.
+ */
+export class BundlerNamespace {
+  /** @internal */
+  constructor(private readonly config: AlchemyConfig) {}
+
+  /**
+   * Sends a User Operation
+   *
+   * Sends a user operation to the provided bundler client evm network, and
+   * returns a user operation hash on successful submission.
+   *
+   * @param userOperation - The user operation.
+   * @public
+   */
+  async sendUserOperation(
+    userOperation: UserOperationRequest
+  ): Promise<string> {
+    const provider = await this.config.getProvider();
+    return provider._send(
+      'eth_sendUserOperation',
+      [userOperation],
+      'sendUserOperation'
+    );
+  }
+
+  /**
+   * Get a User Operation by Hash
+   *
+   * This method returns the user operation object based on the
+   * provided user operation hash.
+   *
+   * @param userOpHash - The hash of the user operation.
+   * @public
+   */
+  async getUserOperationByHash(userOpHash: string): Promise<UserOperation> {
+    const provider = await this.config.getProvider();
+    return provider._send(
+      'eth_getUserOperationByHash',
+      [userOpHash],
+      'getUserOperationByHash'
+    );
+  }
+
+  /**
+   * Get Client Supported Entry Points
+   *
+   * Returns an array of the entry point addresses that are
```

Comment:
can you explain what an entry point address is // used for? where do you pass the return of this method into?